src/test1.js

```diff
@@ -54,6 +56,7 @@ Pet.prototype.feed = function() {
 }
 
 Pet.prototype.checkUp = function() {
+    // consider multiple if statements here
```

Comment:
Here you could instead have multiple if statements and a return at the bottom if neither gets hit:

```js
if (option === 'one') {
  return 'chose option 1';
}

if (option === 'two') {
  return 'chose option 2';
}

return 'chose other option';
```